CI.md

```diff
@@ -17,7 +17,7 @@ For that, we use a [custom docker image](Dockerfile.Colab) in the `linux_glibc_2
 The steps made to build and push that custom image were as follows:
 
 ```sh
-$ docker build -t bytewax/glib-2.27-builder:v0 -f Dockerfile.Colab .
+$ docker build -t bytewax/glib-2.27-builder:v1 -f Dockerfile.Colab .
```

Comment:
Did the image change to add more Python versions? Was that done somewhere else?

Reply:
yes, here: https://github.com/bytewax/bytewax/commit/e81f6a464ae359e51455bd872f57c8b313847352